lib/src/widget/settings/data/settings_tile_action.dart

```diff
@@ -64,6 +66,7 @@ class SettingsTileActionCircle extends SettingsTileAction {
       ];
 }
 
+@Deprecated('So far there is no useCases, but later it might come back')
```

Comment:
💅 Personally I would avoid colloquial language in such descriptions, i.e.: `Old Style Settings, might be removed`

Reply:
updated